### ISSUE TITLE
[Bug - Horizontal] Perfectly functional timelines, but countless notifications of "no date found for ___.md" (for each event), everytime I open the timeline note.

### ISSUE THREAD

**Issue by samuelcasolli** [seanlowe/obsidian-timelines#68]
First I want to thank you so much for this plugin, you saved my life, 
I absolutely love this, is the best timeline plugin that exists, I use everyday, for basically everything. 
And before you, I was having to use some old version of obsidian, only to not lose the Darakah's functioning timeline (that stopped working on the updates of obsidian). Thanks to you, now I can use the updated version of obsidian, and still be able to use timelines

But does exist any way to make those notifications not appear everytime I open a timeline note? Or is it normal?
![image](https://github.com/seanlowe/obsidian-timelines/assets/79317305/783b1fd3-5e60-46a5-a705-654dbb7eb94f)

I tried to check, if I writed something wrong (despite all the timelines appearing and functioning perfectly... On the perfect dates I set...) I tought that maybe I was writing the dates wrong somehow, maybe something related to the dates was missing... I tried including the hours... Tried including other optional sections that I've seen across the tutorials, in those event notes...
But I couldn't find anything that I did, that was wrong comparing to the tutorials... And the notifications persist...
![image](https://github.com/seanlowe/obsidian-timelines/assets/79317305/fb534673-e906-4759-8766-ed07651b689c)

And this happens with all event notes... It doesnt matter if they are "range", or "dot"...

Is it a bug, or am I doing something wrong?

Sorry for the broke english [🇧🇷](https://emojipedia.org/pt/bandeira-brasil)
Thanks in advance

**Comment by seanlowe** [comment]
Hey, thanks for creating an issue. It's interesting that it's giving you that notice for `Agosto de 2024.md` because the event looks correct and the only time I create a Notice is when there is no start date, as in if someone accidentally deleted it or left it blank, etc. And your event clearly has a start date...

Out of curiosity, what does your timeline block look like?

**Comment by seanlowe** [comment]
How interesting!

I recreated your note to see if I could recreate it, and sure enough!

I get this error in the console:
```js
{
  event: {
    tags: "timeline, goals"
  },
  eventData: null
}
```
My first guess is that I'm not handling the tags string correctly and therefore it gets wonky. I'll look into it!

**Comment by seanlowe** [comment]
This has been fixed in release 2.2.2

**Comment by samuelcasolli** [comment]
Damn, it worked perfectly!!
Thank you so much!!!!
